tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/SdkBuildTasks/Microsoft.Azure.Sdk.Build.Tasks.csproj

```diff
@@ -19,6 +20,8 @@
   <PropertyGroup>
     <TargetFramework>net46</TargetFramework>
     <OutputPath>$(TaskBinaryOutput)</OutputPath>
+    <AssemblyOriginatorKeyFile>D:\myFork\LTSBuild\tools\BuildTools.snk</AssemblyOriginatorKeyFile>
```

Comment:
These are already defined in buildtools.props

Reply:
@weshaggard I think if you refresh you will see that I have removed this, I am not sure how come you are still getting stale code changes.